meta-resin-common/recipes-support/resin-init/resin-init-flasher/resin-init-flasher.service

```diff
@@ -2,11 +2,11 @@
 Description=Resin init flasher service
 Requires=\
     mnt-boot.mount \
```

Comment:
My option would be to have less clutter so to improve direct readability of components. But it's fine either way.